modules/dkan_dataset_groups/dkan_dataset_groups.install

```diff
@@ -22,4 +22,49 @@ function dkan_dataset_groups_install() {
 function dkan_dataset_groups_disable() {
   //Remove field groups from the index.
   features_revert(array('dkan_sitewide_search_db' => array('search_api_index', 'search_api_server')));
-}
\ No newline at end of file
+}
+
+/**
```

Comment:
We should actually be doing all updates in dkan.install, not individual module install files. Move this and the datastore ones to dkan.install (eliminating need for dependencies and extra PRs).